core/workspace_comment/workspace_comment_svg.js

```diff
@@ -269,6 +346,60 @@ Blockly.WorkspaceCommentSvg.prototype.clearTransformAttributes_ = function() {
   Blockly.utils.removeAttribute(this.getSvgRoot(), 'transform');
 };
 
+
+/**
+ * Returns the coordinates of a bounding box describing the dimensions of this
+ * comment.
+ * Coordinate system: workspace coordinates.
+ * @return {!{topLeft: goog.math.Coordinate, bottomRight: goog.math.Coordinate}}
+ *    Object with top left and bottom right coordinates of the bounding box.
+ */
+Blockly.WorkspaceCommentSvg.prototype.getBoundingRectangle = function() {
+  var blockXY = this.getRelativeToSurfaceXY(this);
+  var commentBounds = this.getHeightWidth();
+  var topLeft;
+  var bottomRight;
+  if (this.RTL) {
+    // Width has the tab built into it already so subtract it here.
```

Comment:
These comments refer to the puzzle tabs on blocks, so this is probably not necessary.